tests/integration/native-image/mp-1/src/main/java/io/helidon/tests/integration/nativeimage/mp1/Mp1Main.java

```diff
@@ -191,8 +191,9 @@ private static void testBean(int port, String jwtToken) {
         // + JSON-B
         invoke(collector, "Rest client JSON-B", "json-b", aBean::restClientJsonB);
         // + query params
-        invoke(collector, "Rest client Query param long", "1020", () -> aBean.restClientQuery(1020L));
```

Comment:
I updated the message to reflect they're targeting the wrapper Objects and not the primitive type.